include/phasar/PhasarLLVM/DataFlowSolver/Mono/Solver/InterMonoSolver.h

```diff
@@ -177,17 +176,17 @@ template <typename AnalysisDomainTy, unsigned K> class InterMonoSolver {
   }
 
   void processNormal(std::pair<n_t, n_t> Edge) {
-    std::cout << "Handle normal flow\n";
+    llvm::outs() << "Handle normal flow\n";
```

Comment:
Maybe, we should consider using the logger here

Reply:
@pdschubert you agree? what log levels? should we leave that for a different PR and contribution?